single-pool/cli/src/cli.rs

```diff
@@ -114,6 +132,22 @@ pub struct InitializeCli {
     pub skip_metadata: bool,
 }
 
+#[derive(Clone, Debug, Args)]
+#[clap(group(pool_source_group()))]
+pub struct ReactivateCli {
+    /// The pool to reactivate
+    #[clap(short, long = "pool", value_parser = |p: &str| parse_address(p, "pool_address"))]
+    pub pool_address: Option<Pubkey>,
+
+    /// The vote account corresponding to the pool to reactivate
+    #[clap(long = "vote-account", value_parser = |p: &str| parse_address(p, "vote_account_address"))]
+    pub vote_account_address: Option<Pubkey>,
+
+    // backdoor for testing, theres no reason to ever use this
+    #[clap(long, hide = true)]
+    pub yolo: bool,
```

Comment:
Even though it's a backdoor, we can be clearer about its usage, just in case:
```suggestion
    // backdoor for testing, theres no reason to ever use this
    #[clap(long, hide = true)]
    pub skip_deactivation_check: bool,
```